qa/tasks/cephfs/test_quota.py

```diff
@@ -104,3 +104,50 @@ def test_remote_update_write(self):
         with self.assertRaises(CommandFailedError):
             self.mount_b.write_n_mb("subdir_data/file", 40)
 
+    def test_human_readable_quota_values(self):
+        self.mount_a.run_shell(["mkdir", "subdir"])
+
+        self.assertEqual(self.mount_a.getfattr("./subdir",
+                                               "ceph.quota.max_bytes"), None)
+
+        # kilobytes
```

Comment:
s/kilobytes/kibibytes . We are using 1<<10, so it's not kilobytes.

Reply:
done